gettsim/taxes/calc_taxes.py

```diff
@@ -2,7 +2,7 @@
 import pandas as pd
 
 
-def tax_sched(tax_unit, tb):
+def tax_sched(tax_unit, params, e_st_abzuege_params, soli_st_params, abgelt_st_params):
```

Comment:
Here it probably should be `e_st_params`. It is only separate from `e_st_abzuege_params` because otherwise it would be far too big.

Reply:
Changed!